### PR TITLE
Persistence

### PR DESCRIPTION
**Pull request by egdegd**

### CODE REVIEW

File: Internal/domain/link/linkstorage.go [hunk 1]
```diff
@@ -5,10 +5,10 @@ import (
 )
 
 type Link struct {
-	Link string
+	Link        string
 	ShortenLink string
-	Lifetime time.Duration
-	UserId *string
+	Lifetime    *time.Duration
+	AccountId   *string
```
Comment (M0nsteR) [inline Internal/domain/link/linkstorage.go:11]: Do you really need a pointer here? Function signatures that use AccountId are inconsistent now.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - "8080:8080"
+    volumes:
+      - ./app.rsa:/app.rsa
+      - ./app.rsa.pub:/app.rsa.pub
+      - ./cmd/config/config.postgres.json:/cmd/config/config.postgres.json
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: 123455678
```
Comment (M0nsteR) [inline docker-compose.yml:20]: Wrong password: extra 5 is present. 🧮

**Review by M0nsteR — APPROVED** [review]
Good to go.

File: docker-compose.yml (hunk 2 above)
Comment (egdegd) [reply docker-compose.yml:20]: Our password is more secure than the standard 12345678

File: Internal/domain/link/linkstorage.go (hunk 1 above)
Comment (egdegd) [reply Internal/domain/link/linkstorage.go:11]: Pointer is needed because the user can be anonymous. By default, we assume that the pointer is nil.

Comment (M0nsteR) [reply Internal/domain/link/linkstorage.go:11]: Ok, I need to think it over..

**Pull request merged**